CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+ 
+- Added `signInWithNewPassword` GraphQL API for signing in with a new password.
+
```

Comment:
@henry0715-dev,

Could you take a look at this? It indicates that unnecessary spaces are included, which suggests you may need to check your editor setup, as VS Code shouldn't have done this if it were set up properly.

Reply:
I've completed the modification.
Thank you.